### DISCUSSION TITLE
Enhancing Ruby CDP Ferrum with Chrome Accessibility Features

### DISCUSSION THREAD

**Discussion by pedrocarmona** [rubycdp/ferrum#454] [Ideas]
Goal: Incorporating Chrome accessibility features into the Ruby CDP Ferrum framework. 

This will enable developers to create web applications that are not only functional but also easy to use for people
with disabilities.

Resources:
- https://chromedevtools.github.io/devtools-protocol/tot/Accessibility/
- https://playwright.dev/docs/accessibility-testing

**Comment by oscar-edomex** [comment]
Doing this directly through Ferrum is worth a consideration given https://github.com/dequelabs/axe-core-gems/blob/develop/packages/axe-core-capybara/README.md doesn't appear to be compatible with `cuprite` moving forward.

https://github.com/dequelabs/axe-core-gems/issues/243#issue-1349924057
